docs/src/site/twiki/EntitySLAMonitoring.twiki

```diff
@@ -0,0 +1,25 @@
+---++Falcon Entity SLA Monitoring
+
+Entity SLA monitoring allows you to monitor the entity(process and feed) ,It keeps track of the instances of the entity which are running and stores them in the db.
```

Comment:
"entity(process and feed) ,It" -> "entity (process and feed). It"

I would use "that" instead of "which", but that's a preference.